### PR TITLE
Fix for sending empty emails 3.0

### PR DESCRIPTION
**Pull request by korycins**
I want to merge this change because it fixes an issue that Saleor sends empty emails.

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

File: saleor/plugins/user_email/notify_events.py [hunk 1]
```diff
@@ -187,6 +211,10 @@ def send_fulfillment_confirmation(
         constants.ORDER_FULFILLMENT_CONFIRMATION_DEFAULT_TEMPLATE,
         constants.DEFAULT_EMAIL_TEMPLATES_PATH,
     )
+    print(template)
```
Comment (jakubkuc) [inline saleor/plugins/user_email/notify_events.py:214]: 👮🏻‍♂️

Comment (korycins) [reply saleor/plugins/user_email/notify_events.py:214]: 🚔

**Comment by patrys** [comment]
Task linked: [SALEOR-6173 Saleor sends empty emails](https://app.clickup.com/t/2549495/SALEOR-6173)

**Review by maarcingebala — APPROVED** [review]

**Review by jakubkuc — APPROVED** [review]

**Review by IKarbowiak — APPROVED** [review]

**Review by L3str4nge — APPROVED** [review]

File: saleor/plugins/admin_email/notify_events.py [hunk 2]
```diff
@@ -24,6 +24,9 @@ def send_set_staff_password_email(
         constants.SET_STAFF_PASSWORD_DEFAULT_TEMPLATE,
         constants.DEFAULT_EMAIL_TEMPLATES_PATH,
     )
+    if not template:
+        # Empty template means that we don't want to trigger a given event.
```
Comment (L3str4nge) [inline saleor/plugins/admin_email/notify_events.py:28]: How about to create a `@decorator` for this statement? Not in this scope but in the future as an enhancement?

**Pull request merged**